app/views/orgs/rosters/select_google_classroom.html.erb

```diff
@@ -0,0 +1,38 @@
+<% view = FormView.new(subject: @roster) %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h1 class="site-content-heading">Add students</h1>
+  </div>
+
+  <div class="site-content-body">
+    <div class="markdown-body">
+      <p>Let's set up a roster so you can easily track student progress on your dashboard. You can always come back and complete this step later.</p>
+    </div>
+
+    <div class="my-3">
+      <h2 class="h3 mb-2">Select the Google Classroom roster you want to import</h2>
+
+      <div class="Box overflow-auto">
+        <ul>
+          <% @google_classroom_courses.each do |course| %>
+            <li class="Box-row">
+              <label>
+                <input type="checkbox" class="mr-2" checked="checked" aria-describedby="help-text-for-checkbox">
```

Comment:
Since you can only import (and associate with) one Google Classroom (in the interest of syncing later), I suspect these should be radio buttons.